src/EventLoop/Internal/DriverSuspension.php

```diff
@@ -118,7 +118,7 @@ public function suspend(): mixed
                 }
             }
 
-            throw new \Error('Event loop terminated without resuming the current suspension:' . $info);
+            throw new \Error('Event loop terminated without resuming the current suspension (fiber deadlock):' . $info);
```

Comment:
It doesn't have to be a fiber deadlock. One possible reason is a unreferenced or canceled watcher.